lib/add_to_calendar/version.rb

```diff
@@ -1,3 +1,3 @@
 module AddToCalendar
-  VERSION = "0.4.0"
+  VERSION = "0.4.1"
```

Comment:
I think this is a new feature worthy of a [minor](https://semver.org/) version bump :)

```suggestion
  VERSION = "0.5.0"
```